pkg/cli/searcher.go

```diff
@@ -212,17 +382,37 @@ func (spinner *spinnerState) stopSpinner() {
 	}
 }
 
+type printHeader func(writer io.Writer)
+
 func printImageTableHeader(writer io.Writer) {
-	table := getNoBorderTableWriter(writer)
-	row := []string{"IMAGE NAME",
-		"TAG",
-		"DIGEST",
-		"SIZE",
-	}
+	table := getImageTableWriter(writer)
+	row := make([]string, 4)
+
+	row[colImageNameIndex] = "IMAGE NAME"
+	row[colTagIndex] = "TAG"
+	row[colDigestIndex] = "DIGEST"
+	row[colSizeIndex] = "SIZE"
+
+	table.Append(row)
+	table.Render()
+}
+
+func printCVETableHeader(writer io.Writer) {
+	table := getCVETableWriter(writer)
+	row := make([]string, 3)
+	row[colCVEIDIndex] = "ID"
+	row[colCVESeverityIndex] = "SEVERITY"
+	row[colCVETitleIndex] = "TITLE"
+
 	table.Append(row)
 	table.Render()
 }
 
 const (
-	waitTimeout = 6 * time.Second
+	waitTimeout = httpTimeout + 5*time.Second
+)
+
+var (
+	errInvalidImageNameAndTag = errors.New("cli: Invalid input format. Expected IMAGENAME:TAG")
+	errInvalidImageName       = errors.New("cli: Invalid input format. Expected IMAGENAME without :TAG")
```

Comment:
Any specific reason to create new error here instead of pkg/errors

Reply:
The way I thought of this was:
Putting all reused errors in errors.go to be used from multiple instances. These errors were very "local", for the lack of a better word